spec/factories/tasks.rb

```diff
@@ -5,5 +5,15 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    # FactoryBotのtraitを使用
+    trait :another_task do
+      status {'done'}
```

Comment:
`enum`にはStringではなくSymbolで記載しましょう。ただ文字列を設定している訳でないことが示せます。
```suggestion
      status { :done }
```

Reply:
682135b30f7b582d9f151cc43b033d4d7ca667f7
`Symbol`型に変更